lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    cell_start
+  end
+
+  def reset!
+    cell_start
+  end
+
+  def cell_start
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts div_line
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts div_line
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def div_line
+    "-----------"
+  end
+
+  def position(index)
+    @cells[index.to_i - 1]
+  end
+
+  def full?
+    @cells.any? { |cell| cell == " "} ? false : true
+  end
```

Comment:
I think it's strange that I used so many ternary operators explicitly returning a boolean value. I am trying to track down where I started this pattern and I believe it's some code I took from a previous TTT project and copy + pasted over here. Once I got one test passing, I just used the explicit version again and again throughout my code because it worked...  This tells me that I need to give some focus on the refactor stage of TDD and not be so excited about tests being green.

I prefer your use of the implicit boolean and agree that the #all? best expresses what I'm trying to complete with this #full? method.